uint8 WALL_LEFT=0
uint8 WALL_FRONT=1
uint8 WALL_RIGHT=2
uint8 WALL_UNDEF=3
uint8 wall
float32 dist
